### PR TITLE
Add thepetk as has-admin

### PR DESCRIPTION
**Pull request by thepetk**
Requesting access for has-admin as member of the devfiles team.

### CODE REVIEW

File: components/authentication/base/has-admin.yaml [hunk 1]
```diff
@@ -30,6 +30,8 @@ subjects:
     name: kim-tsao
   - kind: User
     name: maysunfaisal
+  - kind: User
+    name: thepetk
```
[the remark this answers is not in the document]
Comment (thepetk) [reply components/authentication/base/has-admin.yaml:34]: Hey @Michkov ! I'm now member of the redhat-appstudio orgs :)

**Review by kim-tsao — APPROVED** [review]
/lgtm

**Pull request merged**